parser/parser.y

```diff
@@ -191,6 +192,8 @@ import (
 	mediumblobType    "MEDIUMBLOB"
 	mediumIntType     "MEDIUMINT"
 	mediumtextType    "MEDIUMTEXT"
+	member		  "member"
```

Comment:
Should it be "MEMBER" (and "MEMBER OF" in the next line) to keep consistent with others?